ci/Dockerfile

```diff
@@ -10,7 +10,10 @@ ARG PYTHON
 ADD ci/requirements-dev-$PYTHON.yml /
 
 RUN conda env create -q -n ibis-$PYTHON -f /requirements-dev-$PYTHON.yml \
- && conda install conda-build -y -q
+ && conda install -y -q conda-build
+
+# We need to install these for docker
+RUN if [ "${PYTHON}" != "3.4" ]; then conda install -c conda-forge -y -q -n ibis-$PYTHON xorg-libxrender xorg-libxpm; fi
```

Comment:
How about appending those `xorg` dependencies to `requirements-dev-3.4.yaml`?

Reply:
Hm but python-graphviz doesn't exist for 3.4 so it's just adding deps for no reason since these are only required for using graphics in the notebook

Reply:
Currection: I mean appending to each requirements files except for py34. Those packages are required to run the tests, right? 
We might do a `conda install ibis` instead of creating environment from requirements file though.
Or maybe `conda build --test` would be more appropiate, this way the recipe should be maintained only.